utilities/workers/worker_manager.py

```diff
@@ -11,23 +16,111 @@ class WorkerManager:
-        args: Arguments to function.
+        class_args: Arguments to function.
+        input_queues: Input queues.
+        output_queues: Output queues.
+        controller: Worker controller.
+
+        Returns whether the workers were able to be created and the Worker Manager.
         """
+        if count == 0:
+            return False, None
+
+        args = WorkerManager.create_worker_arguments(
+            class_args, input_queues, output_queues, controller
+        )
+
+        workers = []
         for _ in range(0, count):
+            result, worker = WorkerManager.create_single_worker(target, args)
+            if not result:
+                return False, None
+
+            workers.append(worker)
+
+        return True, WorkerManager(
+            workers,
+            # target, class_args, input_queues, output_queues, controller
+        )
+
+    @staticmethod
+    def create_worker_arguments(
+        class_args: tuple,
+        input_queues: "list[queue_proxy_wrapper.QueueProxyWrapper]",
+        output_queues: "list[queue_proxy_wrapper.QueueProxyWrapper]",
+        controller: worker_controller.WorkerController,
+    ) -> tuple:
+        """
+        Creates a tuple containing most arguments for a worker.
+
+        class_args: Class arguments.
+        input_queues: Input queues.
+        output_queues: Output queues.
+        controller: Worker controller.
+
+        Returns a tuple with the arguments.
+        """
+        return class_args + tuple(input_queues) + tuple(output_queues) + (controller,)
+
+    @staticmethod
+    def create_single_worker(target: types.FunctionType, args: tuple) -> Tuple[bool, mp.Process]:
+        """
+        Creates a single worker.
+
+        target: Fuction
+        args: Worker arguments.
+
+        Returns whether a worker was created and the worker.
+        """
+        try:
             worker = mp.Process(target=target, args=args)
-            self.__workers.append(worker)
+        except TypeError:
+            return False, None
+
+        return True, worker
+
+    def __init__(
+        self,
+        workers: "list[mp.Process]",
+        # target: "(...) -> object", #type: ignore
+        # class_args: tuple,
+        # input_queues: "list[queue_proxy_wrapper.QueueProxyWrapper]",
+        # output_queues: "list[queue_proxy_wrapper.QueueProxyWrapper]",
+        # controller: worker_controller.WorkerController,
+    ) -> None:
+        """
+        Constructor.
+        """
+        self.__workers = workers
+        # self.__target = target
+        # self.__class_args = class_args
+        # self.__input_queues = input_queues
+        # self.__output_queues = output_queues
+        # self.__controller = controller
+
+    # def check_and_restart_dead_workers(self) -> None:
+    #     """
+    #     TODO
+    #     """
+    #     for worker in self.__workers:
+    #         if not worker.is_alive():
+    #             #TODO: Implement restarting code.
+    #             pass
```

Comment:
remove commented out code (you can preserve these commits in a separate branch, or go back in the commit history for them, but we don't want them in main)